localstack/services/sqs/resource_providers/aws_sqs_queue.py

```diff
@@ -189,9 +189,14 @@ def update(
             model["QueueUrl"] = sqs.create_queue(QueueName=request.desired_state.get("QueueName"))[
                 "QueueUrl"
             ]
+        elif "QueueUrl" not in model:
+            model["QueueUrl"] = sqs.get_queue_url(QueueName=model["QueueName"])["QueueUrl"]
+
+        if "Arn" not in model:
             model["Arn"] = sqs.get_queue_attributes(
                 QueueUrl=model["QueueUrl"], AttributeNames=["QueueArn"]
             )["Attributes"]["QueueArn"]
+
```

Comment:
Given the sample directly as it came from support. I could reproduce it, the `model` would only contain `QueueName` and `FifoTopic` fields.

Reply:
I agree with @dominikschubert here - the `Arn` and `QueueUrl` fields should _never_ be in `request.desired_state` (`model`) since they cannot be specified in the CloudFormation template.

I think they should always be recomputed, since they both contain the `QueueName` and the `QueueName` must change if the queue changes from fifo to non-fifo or its name changes. 

```suggestion
        else:
            model["QueueUrl"] = sqs.get_queue_url(QueueName=model["QueueName"])["QueueUrl"]

        model["Arn"] = sqs.get_queue_attributes(
            QueueUrl=model["QueueUrl"], AttributeNames=["QueueArn"]
        )["Attributes"]["QueueArn"]

```